DigitalInnovationHub/schema.json

```diff
@@ -27,16 +27,16 @@
           "description": "Property. [URL](https://schema.org/URL). Website of the Digital Innovation Hub."
         },
         "logo": {
+          "type": "string",
+          "description": "Property. [URL](https://schema.org/URL). URL of the logo of the Digital Innovation Hub."
```

Comment:
the markdown cannot be used inside the description of the attribute. The right way to do it is Model:'xxxxx', (see contribution manual. https://bit.ly/contribution_manual )